src/d3a/models/market/two_sided.py

```diff
@@ -277,6 +351,6 @@ def _convert_match_to_residual(match):
                 match = replace(match, bid=bid_trade.residual)
             return match
 
-        matchings[start_index:] = [_convert_match_to_residual(match)
-                                   for match in matchings[start_index:]]
-        return matchings
+        matches[start_index:] = [_convert_match_to_residual(match)
```

Comment:
I think we dont play with fire here, so please revert to `matchings` again. Thanks! :-D

Reply:
We are referring to the plural of match IIUC so it should be matches not matchings (matching is the verb)

Reply:
You are right. First translation that popped up for `matches` was the small wooded thing that you light a fire with. Sorry.